egs/tts/VITS/README.md

```diff
@@ -67,11 +76,16 @@ sh egs/tts/VITS/run.sh --stage 1
 ### Configuration
 
 We provide the default hyparameters in the `exp_config.json`. They can work on single NVIDIA-24g GPU. You can adjust them based on your GPU machines.
+For training multi-speaker TTS model, specify the `n_speakers` according to the number of speakers in your dataset(s) and set `multi_speaker_training` to `true`.
 
-```
-"train": {
-        "batch_size": 16,
-    }
+```json
+  "model": {
```

Comment:
I think 

```json
"model": {
    "n_speakers": 1 //Number of speakers in the dataset(s) used. The default value is 1 if not specified.
  },
```
may be better. Using `0` as default value is strange.

Reply:
For the current model structure, setting `n_speaker=0` as the default value allows for more flexible control over whether to use speaker embedding in a single speaker scenario, as well as the size of the speaker embedding. The specific code can be found [here](https://github.com/open-mmlab/Amphion/blob/1c4a9afd0465f0d1318b65c3268bca0230005206/models/tts/vits/vits_trainer.py#L26) and [here](https://github.com/open-mmlab/Amphion/blob/9a743bed2d50412c6715197abec05ae12d3ed2b5/models/tts/vits/vits.py#L247C12-L247C22).

Of course, setting `n_speaker=1` as the default value also works. But it requires modifications in multiple sections of the code, which comes at a high cost.